lib/bot.js

```diff
@@ -143,6 +143,13 @@ class Bot {
         const prelude = `Command sent from Discord by ${username}:`;
         this.ircClient.say(ircChannel, prelude);
         this.ircClient.say(ircChannel, text);
+      } else if (message.attachments) {
```

Comment:
Maybe it would be smart to check if there actually are any attachments too? I.e.:
`} else if (message.attachments && message.attacments.length) {`.

Makes sure it doesn't bug if a message includes an empty attachments array.